lerna.json

```diff
@@ -1,13 +1,12 @@
 {
-  "lerna": "2.4.0",
   "version": "independent",
   "packages": ["packages/*", "native/packages/*"],
   "command": {
     "bootstrap": {
       "ignore": ["bpk-tether"]
     },
     "publish": {
-      "ignoreChanges": ["bpk-tether"],
+      "ignoreChanges": ["bpk-tether", "package-lock.json"],
```

Comment:
So, does this look at *all* `package-lock.json` files and not just the one in the root?